consul/base.py

```diff
@@ -1780,22 +1789,32 @@ def create(
                 params=params,
                 data=data)
 
-        def destroy(self, session_id, dc=None):
+        def destroy(self, session_id, dc=None, token=None):
             """
             Destroys the session *session_id*
 
+            *token* is an optional `ACL token`_ to apply to this request.
+
             Returns *True* on success.
             """
             params = []
             dc = dc or self.agent.dc
             if dc:
                 params.append(('dc', dc))
+            if token:
```

Comment:
```suggestion
            token = token or self.agent.token
            if token:
```
agent may already have token set. Use it!